superset-frontend/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -125,6 +117,15 @@ class DatasourceControl extends React.PureComponent {
     }));
   }
 
+  handleMenuItemClick({ key }) {
+    if (key === '1') {
```

Comment:
Since we are already here, can you make these keys more human readable and maybe use constants?

Reply:
Done